### PR TITLE
OCPBUGS-14052: Introduce runbook for kube-scheduler-operator alert

### PR DESCRIPTION
**Pull request by ingvagabund**
SSIA

### CODE REVIEW

**Comment by ingvagabund** [comment]
[alerts/cluster-kube-scheduler-operator/OWNERS](https://github.com/openshift/runbooks/pull/120/files#diff-d41f4a6e60d8ec0b24a8326c09bddd3e4f46ee9b84bef6e080c34c55de953ea3) is already part of the PR

File: alerts/cluster-kube-scheduler-operator/KubeSchedulerDown.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# KubeSchedulerDown
+
+## Meaning
+
+[This alert][KubeSchedulerDown] is fired when KubeScheduler
```
Comment (simonpasquier) [inline alerts/cluster-kube-scheduler-operator/KubeSchedulerDown.md:5]:
```suggestion
[This alert][KubeSchedulerDown] is fired when the kube-scheduler pods
```

Comment (ingvagabund) [reply alerts/cluster-kube-scheduler-operator/KubeSchedulerDown.md:5]: We should probably update https://github.com/openshift/cluster-kube-scheduler-operator/blob/master/manifests/0000_90_kube-scheduler-operator_03_servicemonitor.yaml#L49 then.

**Review by jan--f — COMMENTED** [review]
The Mitigation section is a bit short of actual mitigation approach. I realize that this can have a wide variety if root causes and its impossible to give specific hints here, but is there anythings besides "it depends" you could put in there? Maybe a link to a troubleshooting guide in the OpenShift docs or similar? Or whether to focus on `kube-scheduler` or `kube-scheduler-operator` first?

**Comment by jan--f** [comment]
@ingvagabund I understand, I'm merely arguing for a message that sounds less hostile for users who get this alert. They will likely be under stress already since their control plane is not working, they find the runbook and the mitigation message is "it depends".
How about something along the lines of
```
Since this alert indicates that the control-plane is not functioning and is unlikely to self-heal, the manual intervention required depend heavily on the exact scenario. The status and log insights gained from the Diagnosis section provide the needed insight. The kube-scheduler pods need to advance to the Running state in order to mitigate this alert.
```

I understand that this doesn't really add more information. Your explanation actually has some great details that could be in that last section.

**Pull request merged**